packages/icons/package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "https://webstudio.is",
   "scripts": {
     "dev": "tsup --watch",
-    "build": "rm -fr lib tsconfig.tsbuildinfo && tsc",
+    "build": "rm -fr lib tsconfig.tsbuildinfo src/generated && svgo-jsx && node ./svgo-jsx-index.js && tsc",
```

Comment:
this should be a separate build step, one we call only manually for now when we add a new svg icon

Reply:
So you want to commit generated icons?